File: app/src/main/java/com/diversedistractions/vehiclelog/PreferencesActivity.java
package com.diversedistractions.vehiclelog;

import android.os.Bundle;
import android.preference.PreferenceFragment;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class PreferencesActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_preferences);

        getFragmentManager()
                .beginTransaction()
                .add(R.id.preferences_content, new PreferenceSettingsFragment())
                .commit();
    }

    public static class PreferenceSettingsFragment extends PreferenceFragment {
        @Override
        public void onCreate(Bundle savedInstanceState) {
            super.onCreate(savedInstanceState);
            addPreferencesFromResource(R.xml.preferences_settings);
        }
    }
}

File: app/src/main/java/com/diversedistractions/vehiclelog/PreferencesActivity.kt
package com.diversedistractions.vehiclelog

import android.os.Bundle
import android.preference.PreferenceFragment
import androidx.appcompat.app.AppCompatActivity

class PreferencesActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_preferences)
        fragmentManager
                .beginTransaction()
                .add(R.id.preferences_content, PreferenceSettingsFragment())
                .commit()
    }

    class PreferenceSettingsFragment : PreferenceFragment() {
        override fun onCreate(savedInstanceState: Bundle?) {
            super.onCreate(savedInstanceState)
            addPreferencesFromResource(R.xml.preferences_settings)
        }
    }
}